src/schema/v2/sale/index.ts

```diff
@@ -178,6 +178,12 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
         resolve: ({ cascading_end_time_interval }) =>
           cascading_end_time_interval,
       },
+      cascadingcascadingEndTimeIntervalMinutesEndTimeInterval: {
```

Comment:
I think there's a typo? Can this be `cascadingEndTimeIntervalMinutes`?

Reply:
😮

Reply:
![6chm67](https://user-images.githubusercontent.com/1457859/163268781-de49a6fe-f0ad-43cc-823b-494738058bd9.jpeg)

Reply:
Omg @mzikherman!

Reply:
🥇